lib/compiler/test_runner.zig

```diff
@@ -244,24 +243,8 @@ pub fn mainSimple() anyerror!void {
         if (enable_print and print_all) stderr.writeAll("PASS\n") catch {};
         passed += 1;
     }
-    if (enable_print) {
+    if (print_summary) {
```

Comment:
I would imagine that you should check for `enable_print and print_summary` here unless the latter is somehow special and does not require stdout support checked via `enable_print`.

Reply:
True! Give me a couple minutes to re-think the structure, this is a bit of a hack.

Reply:
Re-did this!